src/Tests/VCEL.Test/ToJSCodeTests.cs

```diff
@@ -143,6 +143,7 @@ public void TestJsParser_Datetime(string expr, string expected)
     [TestCase("tan(x)", "(Math.tan(vcelContext.x))")]
     [TestCase("tanh(x)", "(Math.tanh(vcelContext.x))")]
     [TestCase("truncate(x)", "(Math.trunc(vcelContext.x))")]
+    [TestCase("isNaN(x)", "(isNaN(vcelContext.x))")]
```

Comment:
Consider using `Number.isNaN()`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN

Reply:
I remembered this requirement on the ticket and already added support for using `Number.isNaN()` as confirmed in unit test. Reder to the updated screen shot.